tests/sparseml/pytorch/sparsification/pruning/test_modifier_pruning_topkast.py

```diff
@@ -125,66 +125,66 @@ def _test_compression_sparsity_applied():
             assert not modifier.update_ready(epoch, test_steps_per_epoch)
             _test_compression_sparsity_applied()
 
-    @flaky(max_runs=3, min_passes=2)
-    def test_weight_decay(
-        self,
-        modifier_lambda,
-        model_lambda,
-        optim_lambda,
-        test_steps_per_epoch,  # noqa: F811
-    ):
-        modifier = modifier_lambda()
-        model = model_lambda()
-        optimizer = optim_lambda(model)
-        self.initialize_helper(modifier, model)
+    # @flaky(max_runs=3, min_passes=2)
```

Comment:
would `@pytest.mark.skip(reason=....)` make more sense? that way we can just make a one line change to turn the test on/off?